code/modules/research/designs/medical_designs.dm

```diff
@@ -118,6 +118,17 @@
 	build_path = /obj/item/device/extinguisher_refill
 	category = list("Medical Designs")
 
+/datum/design/medibeam
+	name = "Medical Beamgun"
+	desc = "Delivers medical nanites in a focused beam."
+	id = "medi_beam"
+	req_tech = list("combat" = 6, "materials" = 7, "powerstorage" = 8, "programming" = 7, "biotech" = 6)
```

Comment:
The highest obtainable programming (Data Theory) level is 6, using Phazon boards. The exception to this is of course the Experimentor, which can theoretically unlock anything. (Although trying to getting more than one tech level up on the same object has astronomically low odds)

I recommend dropping programming down one level, as Combat 6 (Combat Shotguns) and Bio 6 (Sentience Potion) are tough gates to crack internally.